mix.exs

```diff
@@ -35,6 +36,10 @@ defmodule ETS.MixProject do
     ]
   end
 
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
```

Comment:
I tried to get away with not adding a new directory, but to test acceptors, it seems like the best way is to have a server module with all of the clauses, which can be shared by all the test modules.  This also makes a place for `dead_pid` and `keep_alive`.